it/uk/gov/hmrc/incometaxsubscriptioneligibility/helpers/ControlListConfigTestHelper.scala

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmrc.incometaxsubscriptioneligibility.helpers
+
+import play.api.{Application, Environment, Mode}
+import play.api.inject.guice.GuiceApplicationBuilder
+import uk.gov.hmrc.incometaxsubscriptioneligibility.models.controllist.ControlListParameter
+import uk.gov.hmrc.incometaxsubscriptioneligibility.services.ConvertConfigValuesService
+
+trait ControlListConfigTestHelper {
+
+  class Setup(config: Map[ControlListParameter, Boolean]) {
+    lazy val app: Application = new GuiceApplicationBuilder()
+      .in(Environment.simple(mode = Mode.Dev))
+      .configure(toConfigList(config))
+      .build
+    lazy val convertConfigValuesService: ConvertConfigValuesService = app.injector.instanceOf[ConvertConfigValuesService]
+
+    app.stop()
+  }
+
+  def toConfigList(configValues: Map[ControlListParameter, Boolean]): Map[String, String] = {
+    configValues.map {
+      case (param, isEligible) =>
+        s"control-list.${param.configKey}.eligible" -> isEligible.toString
+    }
+  }
+
+  val allControlListParams: Set[ControlListParameter] = ControlListParameter.getParameterMap.values.toSet
+  val testAllTrue: Map[ControlListParameter, Boolean] = allControlListParams.map(
```

Comment:
Could this just be

`ControlListParameter.getParameterMap mapValues(_ => true)`

Rather than converting a map to a set back to a map again? Or is this to ensure there aren't any duplicates?

Reply:
getParameterMap returns a Map[Int, ControlListParameter] and only the ControlListParameters are needed because they need to then be mapped to booleans